tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -901,18 +901,6 @@ static PyObject * THPVariable_new(PyObject* self, PyObject* args, PyObject* kwar
   END_HANDLE_TH_ERRORS
 }
 
-static PyObject * THPVariable_new_ones(PyObject* self, PyObject* args, PyObject* kwargs)
-{
-  HANDLE_TH_ERRORS
-  if (check_has_torch_function(self)) {
-    return handle_torch_function(self, "new_ones", args, kwargs);
-  }
-  auto& self_ = THPVariable_Unpack(self);
-  OptionalDeviceGuard device_guard(device_of(self_));
-  return THPVariable_Wrap(torch::utils::new_ones(legacyExtractDispatchKey(self_), self_.scalar_type(), args, kwargs));
```

Comment:
Can we delete torch::utils::new_ones as well? It's in torch/csrc/utils/tensor_new.cpp